x/evm/types/msg.go

```diff
@@ -403,3 +404,18 @@ func (m *MsgUpdateParams) ValidateBasic() error {
 
 	return m.Params.Validate()
 }
+
+// ValidateBasic does a sanity check for MsgRegisterPreinstalls
+func (m *MsgRegisterPreinstalls) ValidateBasic() error {
+	if _, err := sdk.AccAddressFromBech32(m.Authority); err != nil {
+		return errorsmod.Wrap(err, "invalid authority address")
```

Comment:
Can also print the authority address here.

Reply:
authority address is always governance module. maybe not needed to print